frontend/src/components/pages/AdminControls/EmployeeDirectoryTable.tsx

```diff
@@ -246,7 +264,11 @@ const EmployeeDirectoryTable = ({
           <EditEmployee
             employee={editingEmployee}
             isOpen={isEditModalOpen}
-            toggleClose={() => setIsEditModalOpen(false)}
+            toggleClose={() => {
+              setIsEditModalOpen(false)
+              getRecords(userPageNum)
```

Comment:
So you should be passing this in as a prop to the component because we don't want this to trigger unless the form is submitted (not just closed). This can be added to `EditEmployee` on line 160 before `toggleClose` is called